.github/workflows/klocwork.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+name: klocwork
+on:
+  schedule:
+    - cron: 0 0 * * *

```

Comment:
Is this job running on a schedule or manually (or both?) I'd think manual runs should suffice.

Reply:
Sure, I'll take out the scheduled run then